src/integrationTest/java/uk/gov/hmcts/reform/pip/data/management/controllers/PublicationTest.java

```diff
@@ -1789,8 +1789,6 @@ void retrieveMetadataWhenVerificationEndpointNotSet() throws Exception {
             exceptionResponse.getMessage().contains(VERIFICATION_HEADER),
             "Verification error not shown in error message"
         );
-        assertTrue(exceptionResponse.getMessage().contains(VERIFICATION_HEADER),
-                   "Verification error not shown in error message");
     }
```

Comment:
We should add some integration tests for the NoMatch logic, if the court doesn't exist

Reply:
as discussed, add integration tests.